packages/components/src/color-palette/style.scss

```diff
@@ -7,10 +7,12 @@
 	padding: $grid-unit-15;
 	font-family: inherit;
 	width: 100%;
+	/*rtl:begin:ignore*/
```

Comment:
Thanks for fixing this. 

It'd be great to add a comment to say why this is ignored. It would help incidental readers of this code understand what the gradient does.

Reply:
Thanks for the review 👍 Comment added.